packages/xdl/src/detach/IosShellApp.js

```diff
@@ -236,8 +243,9 @@ async function configureAndCopyArchiveAsync(args) {
           shell: true,
         }
       );
-    } else if (type === 'archive') {
-      await spawnAsync('/bin/mv', [`${DEFAULT_EXPOKIT_WORKSPACE_NAME}.xcarchive`, output], {
+    } else if (type === 'archive' || type === 'client') {
+      const workspaceName = type === 'archive' ? EXPOKIT_APP : EXPONENT_APP;
+      await spawnAsync('/bin/mv', [`${workspaceName}.xcarchive`, output], {
```

Comment:
I know this code was not added in this pull request, but would it make sense to use `fs.move` (from fs-extra) to make this cross-platform? Or is this code only ever run on macOS?

Reply:
We only run this code on macOS.